File: src/javascriptAPI/lib/fmt/fmt.js
fmt = function(){
    var exports = {}


    exports.Print = function(...any){
        returning_print_buffer +=  String(call_go_fn("fmt", "Print", ...any))
    }
    exports.Printf = function(...any){
        returning_print_buffer +=  String(call_go_fn("fmt", "Printf", ...any))
    }
    exports.Println = function(...any){
        returning_print_buffer +=  String(call_go_fn("fmt", "Println", ...any))
    }
    exports.Sprint = function(...any){
        return  String(call_go_fn("fmt", "Print", ...any))
    }
    exports.Sprintf = function(...any){
        return String(call_go_fn("fmt", "Printf", ...any))
    }
    exports.Sprintln = function(...any){
        return  String(call_go_fn("fmt", "Println", ...any))
    }
    return exports
}()

File: src/javascriptAPI/lib/fmt/fmt.go
package fmtjs

import "fmt"

func JsHandle(args ...string) string {
	switch args[0] {
	case "Print":
		a := []interface{}{}
		for _, v := range args[1:] {
			a = append(a, v)
		}
		return fmt.Sprint(a...)
	case "Printf":
		a := []interface{}{}
		for _, v := range args[2:] {
			a = append(a, v)
		}
		return fmt.Sprintf(args[1], a...)
	case "Println":
		a := []interface{}{}
		for _, v := range args[1:] {
			a = append(a, v)
		}
		return fmt.Sprintln(a...)

	}
	return ""
}
